src/components/shared/TreeView.js

```diff
@@ -16,6 +16,8 @@ import type { IndexIntoCallNodeTable, Node } from '../../types/profile-derived';
 import type { CallTree } from '../../profile-logic/call-tree';
 import type { IconWithClassName } from '../../types/reducers';
 
+type RegExpResult = null | ({ index: number, input: string } & string[]);
```

Comment:
This is fine for here now, but should this be hoisted to be a global type?

Reply:
Actually I think this should go into Flow... See https://github.com/facebook/flow/issues/4099 and https://github.com/facebook/flow/issues/3554

Reply:
I should reference these bugs here.